pages/causes/create/success/[slug].tsx

```
@@ -0,0 +1,16 @@
+import { useRouter } from 'next/router';
+import { useQuery } from '@apollo/client';
+import { GET_CAUSE_BY_SLUG } from '@/apollo/gql/gqlCauses';
```

ESLint

[error] GET_CAUSE_BY_SLUG not found in '@/apollo/gql/gqlCauses'

(import/named)